Test/Unit/Service/Product/FacetBuilderTestable.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Test\Unit\Service\Product;
+
+use Algolia\AlgoliaSearch\Service\Product\FacetBuilder;
+
+class FacetBuilderTestable extends FacetBuilder
```

Comment:
Just wanted to check why is this needed ?

Reply:
Ah! This is purely so I could perform unit tests on `protected` methods that I did not want to expose in the API.

Reply:
Ah makes sense ! Very smart !